pkg/extensions/main.go

```diff
@@ -25,7 +25,6 @@ import (
 	_ "github.com/stretchr/testify/require"
 	_ "github.com/vectordotdev/go-datemath"
 	_ "golang.org/x/time/rate"
-	_ "google.golang.org/genproto/googleapis/cloud/kms/v1"
```

Comment:
Just checking this isn't causing an issue for enterprise?

Reply:
this was removed from `main` with no further comment so I assume no. The library has been deprecated, that's why I had to remove it, the linter complained.